src/Microsoft.ML.TimeSeries/Microsoft.ML.TimeSeries.csproj

```diff
@@ -5,6 +5,10 @@
     <IncludeInPackage>Microsoft.ML.TimeSeries</IncludeInPackage>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML.Mkl.Redist" Version="1.0.0" />
```

Comment:
Isn't this needed in `pkg\Microsoft.ML.TimeSeries\Microsoft.ML.TimeSeries.nupkgproj` instead?

Reply:
Instead of `PackageReference`, shouldn't it be a `ProjectReference`?